src/sentry/static/sentry/app/views/ruleEditor/index.jsx

```diff
@@ -158,9 +158,7 @@ const RuleEditor = createReactClass({
 
             {this.hasError('conditions') && (
               <p className="error">
-                {t(
-                  'Ensure at least one condition is enabled and all required fields are filled in.'
-                )}
+                {this.state.error['conditions'][0]}
```

Comment:
possibly worth commenting this is a hack?  will errors always have conditions with >0 items

Reply:
The `hasError` check implies that if the field is missing from the errors object not to mark it with errors.

https://github.com/getsentry/sentry/blob/3d32be4a05b14e94590e3b85188245120bfac6ae/src/sentry/static/sentry/app/views/ruleEditor/index.jsx#L102

So I think it's safe to assume it will have errors.

I consider this part a bit less of a hack, I just didn't want to render all of validation errors. Think it's still worth commenting?